elements/lisk-transactions/src/0_transfer_transaction.ts

```diff
@@ -260,4 +260,16 @@ export class TransferTransaction extends BaseTransaction {
 
 		return errors;
 	}
+
+	// tslint:disable:next-line: prefer-function-over-method no-any
+	protected assetDbRead(raw: any): object | undefined {
```

Comment:
We don't use `db` term anywhere in the project anymore. Now we use `storage` everywhere. Please refactor all the code to update reference of `db` with `storage`

Reply:
this is intended for 2.0, and we don't want to refactor this now.
Also, this method will be removed in 3.0.